{"uuid": "6b36d2b4-6e76-4e90-9520-1d604d40066b", "children": ["9e833d4f-ba7f-4ed7-b160-906df930154a"], "befores": [{"name": "init_driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: No matching capabilities found\nStacktrace:\n\tGetHandleVerifier [0x00007FF748EA5D42+55138]\n\tMicrosoft::Applications::Events::ILogConfiguration::operator* [0x00007FF748E321C2+219010]\n\t(No symbol) [0x00007FF748C63407]\n\t(No symbol) [0x00007FF748CD1400]\n\t(No symbol) [0x00007FF748CD0B83]\n\t(No symbol) [0x00007FF748CD2251]\n\t(No symbol) [0x00007FF748CC8A73]\n\t(No symbol) [0x00007FF748C9E53E]\n\t(No symbol) [0x00007FF748C9D6FC]\n\t(No symbol) [0x00007FF748C9E0F1]\n\tMicrosoft::Applications::Events::EventProperty::to_string [0x00007FF74905ED2C+1067068]\n\t(No symbol) [0x00007FF748D133C6]\n\t(No symbol) [0x00007FF748DACFAC]\n\t(No symbol) [0x00007FF748DA4978]\n\tMicrosoft::Applications::Events::EventProperty::to_string [0x00007FF74905DC75+1062789]\n\tMicrosoft::Applications::Events::ILogConfiguration::operator* [0x00007FF748E3BBB1+258417]\n\tMicrosoft::Applications::Events::ILogConfiguration::operator* [0x00007FF748E36AB4+237684]\n\tMicrosoft::Applications::Events::ILogConfiguration::operator* [0x00007FF748E36BEF+237999]\n\tMicrosoft::Applications::Events::ILogConfiguration::operator* [0x00007FF748E2D6A1+199777]\n\tBaseThreadInitThunk [0x00007FF8156453E0+16]\n\tRtlUserThreadStart [0x00007FF8158E485B+43]\n\n", "trace": "  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\_pytest\\fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\GIT_Projects\\CAW\\tests\\conftest.py\", line 14, in init_driver\n    web_driver = webdriver.Chrome(service=service_obj)\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 49, in __init__\n    super().__init__(\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 54, in __init__\n    super().__init__(\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 206, in __init__\n    self.start_session(capabilities)\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 291, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1709059325272, "stop": 1709059329715}], "start": 1709059325272, "stop": 1709059330388}